docs/Tutorial_MPDist.ipynb

```diff
@@ -0,0 +1,309 @@
+{
```

Comment:
<p>Consider replacing the for loop with this:</p><pre class="ql-syntax" spellcheck="false">data = loadmat('MaryBethAnhLisa_data.mat')['XSsY4']
dfs = {data[1][i][0]:pd.DataFrame(data[0][i].flatten()) for i in range(data[0].shape[0])}
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/433/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>I think the for-loop is easier to read</p>

Reply:
Yea I agree, the dictionary comprehension is far from readable. Maybe I was just challenging myself to write a one liner.....

In any case, the final version should look like neither since we'll upload the datasets to Zenodo. Maybe you could do the data wrangling locally and THEN upload the cleaned dataset to Zenodo, so that on the final tutorial, it's just a matter of indexing a single dimension.

Reply:
<p>Yeah, cleaning it first sounds like a good idea, I will fix this later</p>

Reply:
Is using Zenodo something I can do for this something I can do? @seanlaw

Reply:
<p>@asifmalik I can help you with that. I'm thinking that we'll just create 6 separate files (one for each name)?</p>